stdlib/public/core/CMakeLists.txt

```diff
@@ -129,8 +129,8 @@ split_embedded_sources(
   EMBEDDED RandomAccessCollection.swift
   EMBEDDED Range.swift
   EMBEDDED RangeReplaceableCollection.swift
-    NORMAL RangeSet.swift
-    NORMAL RangeSetRanges.swift
+  EMBEDDED RangeSet.swift
+  EMBEDDED RangeSetRanges.swift
```

Comment:
Should we also include `DiscontiguousSlice.swift` at the same time or is that not compatible with embedded swift? That allows slicing collections with `RangeSet`s.

Reply:
I didn't test yet on that one; I can check

Reply:
yes, that can be done in a seperate commit